src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costEveryDay = 40;
+  let totalcost = costEveryDay * days;
+
+  if (days >= 7) {
+    totalcost -= 50;
```

Comment:
Consider extracting the discount values (50 and 20) into variables with meaningful names, e.g. 'longTermDiscount' and 'shortTermDiscount'.